src/site/content/en/fast/measure-performance-with-the-rail-model/index.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Measure performance with the RAIL model
+authors:
+  - addyosmani
+  - developit
+description: |
+  RAIL model enables designers and developers to reliably target the performance optimization work that has the highest impact on user experience. Learn what goals and guidelines the RAIL model sets out and which tools you can use to achieve them.
+date: 2020-06-05
+---
+
+**RAIL** is a **user-centric** performance model that provides a structure for
+thinking about performance. The model breaks down the user's experience into key
+actions (for example, tap, scroll, load) and helps you define performance goals
+for each of them.
+
+RAIL stands for four distinct aspects of web app life cycle: response,
+animation, idle, and load. Users have different performance expectations for
+each of these contexts, so performance goals are defined based on the context
+and [UX research on how users perceive
+delays](https://www.nngroup.com/articles/response-times-3-important-limits/).
+
+<figure class="w-figure">
+  <img src="./rail.png" alt="The 4 parts of the RAIL performance model: response, animation, idle, and load.">
+  <figcaption class="w-figcaption">Figure 1. The 4 parts of the RAIL performance model
+</figcaption>
+</figure>
+
+
+## Focus on the user
+
+Make users the focal point of your performance effort. The table below describes
+key metrics of how users perceive performance delays:
+
+<table class="w-table-wrapper">
+  <thead>
+  User perception of performance delays
+  </thead>
+  <tr>
+    <td>0 to 16&nbsp;ms</td>
+    <td>Users are exceptionally good at tracking motion, and they dislike it when animations aren't smooth. They perceive animations as smooth so long as 60 new frames are rendered every second. That's 16&nbsp;ms per frame, including the time it takes for the browser to paint the new frame to the screen, leaving an app about 10&nbsp;ms to produce a frame.</td>
+  </tr>
+  <tr>
+    <td>0 to 100&nbsp;ms</td>
+    <td>Respond to user actions within this time window and users feel like the result is immediate. Any longer, and the connection between action and reaction is broken.</td>
+  </tr>
+  <tr>
+    <td>100 to 1000&nbsp;ms</td>
+    <td>Within this window, things feel part of a natural and continuous progression of tasks. For most users on the web, loading pages or changing views represents a task.</td>
+  </tr>
+  <tr>
+    <td>1000&nbsp;ms or more</td>
```

Comment:
This breaks into new line my laptop, is it okay to sprinkle in some more `&nbsp;`?

Reply:
Yes